.github/scripts/summarize_mvcc_closeout.py

```
@@ -0,0 +1,183 @@
+def parse_resources(path: Path, expected_samples: int) -> dict[str, float]:
+    text = path.read_text(encoding="utf-8", errors="replace")
+    rss = [int(value) for value in RSS_RE.findall(text)]
+    user = [float(value) for value in USER_RE.findall(text)]
+    system = [float(value) for value in SYSTEM_RE.findall(text)]
+    expected_invocations = expected_samples * 2
+    counts = (len(rss), len(user), len(system))
+    if counts != (expected_invocations,) * 3:
+        raise ValueError(
+            f"{path}: expected {expected_invocations} complete /usr/bin/time "
+            f"invocations, got rss/user/system={counts}"
+        )
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)